test/testinput/bufr_ncep_rtma_prepbufr_adpupa.yaml

```diff
@@ -0,0 +1,227 @@
+# (C) Copyright 2024 NOAA/NWS/NCEP/EMC
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+
+observations:
+  - obs space:
+      name: bufr
+
+      obsdatain: "testinput/rtma_ADPUPA.prepbufr"
+
+      exports:
+        group_by_variable: prepbufrDataLevelCategory
+        variables:
+          # MetaData
+          timestamp:
+            timeoffset:
+              timeOffset: "*/PRSLEVEL/DRFTINFO/HRDR"
+              transforms:
+                - scale: 3600
+              referenceTime: "2020-11-01T12:00:00Z"
+          stationIdentification:
+            query: "*/SID"
+          stationElevation:
+            query: "*/ELV"
+            type: float
+          prepbufrDataLevelCategory:
+            query: "*/PRSLEVEL/CAT"
+          longitude:
+            query: "*/PRSLEVEL/DRFTINFO/XDR"
+          latitude:
+            query: "*/PRSLEVEL/DRFTINFO/YDR"
+          height:
+            query: "*/PRSLEVEL/Z___INFO/Z__EVENT{1}/ZOB"
+            type: float
+          pressure:
+            query: "*/PRSLEVEL/P___INFO/P__EVENT{1}/POB"
+            type: float
+            transforms:
+              - scale: 100
+
+          # ObsValue
+          airTemperature:
+            query: "*/PRSLEVEL/T___INFO/T__EVENT{1}/TOB"
+            transforms:
+              - offset: 273.15
+          dewpointTemperature:
+            query: "*/PRSLEVEL/Q___INFO/TDO"
+            transforms:
+              - offset: 273.15
+          virtualTemperature:
+            query: "*/PRSLEVEL/T___INFO/TVO"
+            transforms:
+              - offset: 273.15
+          windEastward:
+            query: "*/PRSLEVEL/W___INFO/W__EVENT{1}/UOB"
+          windNorthward:
+            query: "*/PRSLEVEL/W___INFO/W__EVENT{1}/VOB"
+          specificHumidity:
+            query: "*/PRSLEVEL/Q___INFO/Q__EVENT{1}/QOB"
+            type: float
+            transforms:
+              - scale: 0.000001
+
+          # QualityMark
+          airTemperatureQualityMarker:
+            query: "*/PRSLEVEL/T___INFO/T__EVENT{1}/TQM"
+          pressureQualityMarker:
+            query: "*/PRSLEVEL/P___INFO/P__EVENT{1}/PQM"
+          windQualityMarker:
+            query: "*/PRSLEVEL/W___INFO/W__EVENT{1}/WQM"
+          specificHumidityQualityMarker:
+            query: "*/PRSLEVEL/Q___INFO/Q__EVENT{1}/QQM"
+          heightQualityMarker:
+            query: "*/PRSLEVEL/Z___INFO/Z__EVENT{1}/ZQM"
+
+          # ObsError
+          airTemperatureObsError:
+            query: "*/PRSLEVEL/T___INFO/T__BACKG/TOE"
+            transforms:
+              - offset: 273.15
```

Comment:
Double check this. Because one degree Kelvin = one degree Celsius, this shouldn't have to change.

Reply:
@nicholasesposito thanks, fixed.